integration/test/utils/ws-promise.ts

```diff
@@ -1,26 +1,53 @@
 import * as WebSocket from 'ws';
 
+export const createConnection = (
```

Comment:
If there is a better way to write all of this, please, let me know. I spent a full day on getting this working properly